src/util/refs.ts

```diff
@@ -0,0 +1,41 @@
+'use strict';
```

Comment:
There should be original copyright header and I believe we should put link to original repository.

Reply:
Updated.